server.js

```diff
@@ -1,13 +1,22 @@
+// using mLab/d2rd/notes db.collection(notes)
 const express = require('express');
 const helmet = require('helmet');
 const mongoose = require('mongoose');
+// const Schema = mongoose.Schema;
+const Axios = require("axios");
```

Comment:
don't need axios in backend :)

Reply:
This is because Axios handles HTTP requests between front end and backend?  Please confirm my understanding:
When using Axios the syntax is `axios.get`, `axios.post` etc. 
If I use express.router() the convention would be `router.get`, `router.post` etc.
If I assign the variable `server` to = `express.router()` I could user `server.get`, `server.post` etc.?